data_steward/validation/metrics/required_labs.py

```diff
@@ -15,6 +21,75 @@
 MEASUREMENT_CONCEPT_SETS_TABLE = '_measurement_concept_sets'
 MEASUREMENT_CONCEPT_SETS_DESCENDANTS_TABLE = '_measurement_concept_sets_descendants'
 
+ROW_COUNT_QUERY = JINJA_ENV.from_string("""
+SELECT table_id, row_count
+FROM `{{project_id}}.{{dataset_id}}.__TABLES__`""")
+
+
+def check_and_copy_tables(project_id, dataset_id):
+    """
+    Will check that all the required tables exist and if not, they will be created
+    or copied from another table.
+
+    :param project_id: Project where the dataset resides
+    :param dataset_id: Dataset where the required lab table needs to be created
+    :return: None
+    """
+
+    descendants_table_name = f'{project_id}.{dataset_id}.{MEASUREMENT_CONCEPT_SETS_DESCENDANTS_TABLE}'
+    concept_sets_table_name = f'{project_id}.{dataset_id}.{MEASUREMENT_CONCEPT_SETS_TABLE}'
+
+    client = bq.get_client(project_id)
+    dataset = client.dataset(dataset_id)
+    vocab_dataset = client.dataset(common.VOCABULARY_DATASET)
+
+    # concept table and concept ancestor table source tables
+    concept_source_table = vocab_dataset.table(common.CONCEPT)
+    concept_ancestor_source_table = vocab_dataset.table(common.CONCEPT_ANCESTOR)
+
+    # concept table and concept ancestor table destination tables
+    concept_dest_table = dataset.table(common.CONCEPT)
+    concept_ancestor_dest_table = dataset.table(common.CONCEPT_ANCESTOR)
+
+    # query will check the row counts of each table in the specified dataset
+    row_count_query = ROW_COUNT_QUERY.render(project_id=project_id,
+                                             dataset_id=dataset_id)
+
+    results_dataframe = client.query(row_count_query).to_dataframe()
+
+    # checks if CONCEPT and CONCEPT_ANCESTOR tables exist, if they don't, they are copied from the
+    # CONCEPT and CONCEPT_ANCESTOR tables in common.VOCABULARY
+    if common.CONCEPT not in (results_dataframe['table_id']).any():
+        client.copy_table(concept_source_table, concept_dest_table)
+    if common.CONCEPT_ANCESTOR not in (results_dataframe['table_id']).any():
```

Comment:
nit: You can use `.values` to treat it like a Python List. E.g
```suggestion
    if common.CONCEPT_ANCESTOR not in results_dataframe['table_id']).values:
```